include/json/value.h

```diff
@@ -649,7 +649,7 @@ Json::Value obj_value(Json::objectValue); // {}
 
   class Comments {
   public:
-    Comments() = default;
+    Comments();
```

Comment:
Why remove the default constructor? Now you have a linker error due to Comments::Comments being undefined.